### PR TITLE
[stable/nginx-ingress] Add env variables to defaultBackend deployment

### PR DESCRIPTION
**Pull request by iwalmsley**
Signed-off-by: Iain Walmsley <iain.walmsley@gmail.com >

#### What this PR does / why we need it:
We deploy a default back-end image that is parameterized using environmental variables - this change allows this. 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/nginx-ingress/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.16.0
+version: 1.16.1
```
Comment (ChiefAlexander) [inline stable/nginx-ingress/Chart.yaml]: As this is a new feature lets bump the minor version here. `1.17.0`
After that bump this change looks good!

**Review by ChiefAlexander — CHANGES_REQUESTED** [review]

File: stable/nginx-ingress/Chart.yaml (hunk 1 above)
Comment (iwalmsley) [reply stable/nginx-ingress/Chart.yaml]: Bumped that now - thanks!

**Comment by ChiefAlexander** [comment]
Thanks for the PR!
/lgtm

**Pull request merged**